### DISCUSSION TITLE
convert timespan to second

### DISCUSSION THREAD

**Discussion by MohMehrnia** [ravendb/ravendb#18372] [General]
i can converted string 00:20:25.14 by Ticks() to numbers but how do convert to second

**Comment by ppekrol** [comment]
Can you give us a bit more context? In what operation you are interested to convert that to seconds? Patching? Projections? Indexing?

**Comment by ayende** [comment]
Assuming that this is a `TimeSpan` value, you should be able to call `TotalSeconds`, no?
